src/java.desktop/macosx/native/libawt_lwawt/awt/CClipboard.m

```diff
@@ -174,6 +174,40 @@ - (BOOL) checkPasteboardWithoutNotification:(id)application {
 JNI_COCOA_EXIT(env);
 }
 
+/*
+ * Class:     sun_lwawt_macosx_CClipboard
+ * Method:    writeFileObjects
+ * Signature: ([BJ)V
+*/
+JNIEXPORT void JNICALL Java_sun_lwawt_macosx_CClipboard_writeFileObjects
+(JNIEnv *env, jobject inObject, jbyteArray inBytes, jlong inFormat)
+{
```

Comment:
This native method takes a long, but I don't see that in the Java code ??

Reply:
Thanks for the catch, it's not needed anymore. I'll update the patch.